ai-guide/src/modules/home/ui/views/home-view.tsx

```
@@ -0,0 +1,37 @@
+              },
+            },
+          })
+        }
+      >
+        {" "}
+        Sign out
+      </Button>
+    </div>
+  );
+
+  return <div>Home Page</div>;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)